buildSrc/src/main/kotlin/ProjectConfiguration.kt

```diff
@@ -0,0 +1,19 @@
+const val targetVersion = "2.1.1"
+const val kotlin_version = "1.3.61"
```

Comment:
Those two constants are referenced in a lot of groovy `build.gradle`, requiring weird `ProjectConfigurationKt.kotlin_version` syntax. I propose to include `ext.set("kotlin_version", kotlin_version)` in root `build.gradle.kts` for compatibility purposes

Reply:
Y, when migrate other configs ProjectConfigurationKt was not required. And now its better performance instead strings map.